DSharpPlus/Clients/DiscordClient.cs

```diff
@@ -694,6 +694,20 @@ public Task<DiscordGuildTemplate> GetTemplateAsync(string code)
         public Task<DiscordApplicationCommand> GetGlobalApplicationCommandAsync(ulong commandId) =>
             this.ApiClient.GetGlobalApplicationCommandAsync(this.CurrentApplication.Id, commandId);
 
+        /// <summary>
+        /// Gets a global application command by its name.
+        /// </summary>
+        /// <param name="commandName">The name of the command to get.</param>
+        /// <returns>The command with the name.</returns>
+        public async Task<DiscordApplicationCommand> GetGlobalApplicationCommandAsync(string commandName)
+        {
+            foreach (var command in await this.GetGlobalApplicationCommandsAsync())
```

Comment:
Generally we use linq in the library but I'm fairly sure this is a bit faster. Just letting ya know anyway :)